src/Xamarin.Android.Build.Tasks/Microsoft.NET.Sdk.Android/WorkloadManifest.targets

```diff
@@ -2,7 +2,7 @@
   <Import Project="Sdk.targets" Sdk="Microsoft.Android.Sdk"
       Condition=" '$(TargetPlatformIdentifier)' == 'android' " />
   <Import Project="Sdk.targets" Sdk="Microsoft.Android.Sdk.BundleTool"
-      Condition=" '$(AndroidPackageFormat)' == 'aab' " />
+      Condition=" '$(AndroidPackageFormat)' == 'aab' Or $(AndroidPackageFormats.Contains ('aab')) " />
```

Comment:
I was talking about this with @jonpryor, I think you can just completely remove the `Condition` here and always import it.